packages/opentelemetry-plugin-grpc-js/src/server/patchServer.ts

```diff
@@ -72,6 +73,15 @@ export function patchServer(
           ) {
             const self = this;
 
+            if (shouldNotTraceServerCall.call(plugin, call.metadata, name)) {
```

Comment:
Use of call here is unnecessary and confusing. Making the function a private method of the class would be much better.

```suggestion
            if (plugin.shouldNotTraceServerCall(call.metadata, name)) {
```